### ISSUE TITLE
ResolvePackageNotFound: 

### ISSUE THREAD

**Issue by msabers** [RomeroBarata/fidgetyfind#2]
Hello,
Thanks for sharing the code. When I try to create the environment based on the environment.yml file, I receive the below error. 
Could you please help me to solve it?

ResolvePackageNotFound: 
- _openmp_mutex==5.1=1_gnu
- libgfortran-ng==11.2.0=h00389a5_1
- xz==5.4.2=h5eee18b_0
- numpy==1.21.2=py39h20f2e39_0
- pip==23.3=py39h06a4308_0
- pytz==2023.3.post1=py39h06a4308_0
- tk==8.6.12=h1ccaba5_0
- libgomp==11.2.0=h1234567_1
- setuptools==68.0.0=py39h06a4308_0
- bottleneck==1.3.5=py39h7deecbd_0
- ld_impl_linux-64==2.38=h1181459_1
- python==3.9.7=h12debd9_1
- packaging==23.1=py39h06a4308_0
- zlib==1.2.13=h5eee18b_0
- libgcc-ng==11.2.0=h1234567_1
- intel-openmp==2021.4.0=h06a4308_3561
- libffi==3.3=he6710b0_2
- openssl==1.1.1w=h7f8727e_0
- mkl_fft==1.3.1=py39hd3c417c_0
- mkl==2021.4.0=h06a4308_640
- readline==8.2=h5eee18b_0
- scipy==1.7.3=py39h6c91a56_2
- mkl_random==1.2.2=py39h51133e4_0
- fftw==3.3.9=h27cfd23_1
- sqlite==3.41.2=h5eee18b_0
- ncurses==6.4=h6a678d5_0
- libgfortran5==11.2.0=h1234567_1
- libstdcxx-ng==11.2.0=h1234567_1
- mkl-service==2.4.0=py39h7f8727e_0
- numpy-base==1.21.2=py39h79a1101_0
- ca-certificates==2023.08.22=h06a4308_0
- wheel==0.41.2=py39h06a4308_0
- numexpr==2.8.4=py39he184ba9_0
- pandas==1.4.2=py39h295c915_0

**Comment by RomeroBarata** [comment]
Hi msabers,

Quick question: what is your operating system?

**Comment by msabers** [comment]
> Hi msabers,
> 
> Quick question: what is your operating system?

thanks for your quick response,
I use Win 10

**Comment by RomeroBarata** [comment]
That is probably the issue. I use a Unix-based OS (Ubuntu 20.04) and so some of the libraries that come along are Unix specific.

But, if you create the environment from scratch, the Anaconda solver should pick the correct (Windows equivalent) packages for your system. I believe this should create an environment that works for you:

```bash
conda create --name fidgetyfind python=3.9 numpy=1.21.2 scipy=1.7.3 pandas=1.4.2
conda activate fidgetyfind
pip install opencv-python==4.5.1.48
```

Let me know how it goes!

**Comment by msabers** [comment]
> That is probably the issue. I use a Unix-based OS (Ubuntu 20.04) and so some of the libraries that come along are Unix specific.
> 
> But, if you create the environment from scratch, the Anaconda solver should pick the correct (Windows equivalent) packages for your system. I believe this should create an environment that works for you:
> 
> ```shell
> conda create --name fidgetyfind python=3.9 numpy=1.21.2 scipy=1.7.3 pandas=1.4.2
> conda activate fidgetyfind
> pip install opencv-python==4.5.1.48
> ```
> 
> Let me know how it goes!

Thanks for your response,
Unfortunately, these solutions didn't work for me.
Anaconda Slover couldn't solve it and showed an error and asked me to report it to the server.
Creating the environment with the above code didn't work again, it needed different dependencies, I tried to install them one by one. But after installing Tensorflow another error appeared.
I tried to run the code on Ubuntu (WSL) and this is the last error:
OSError: Unable to open file (unable to open file: name = './model/keras/model.h5', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)

![Untitled](https://github.com/RomeroBarata/fidgetyfind/assets/129945999/e5a8703f-556a-4349-8666-26fa581bb116)

**Comment by RomeroBarata** [comment]
Hi msabers,

Just to clarify, the `openpose_keras` and `fidgetyfind` are two separate environments. If you try to install the packages in both you will likely have conflicts.

For the `openpose_keras` environment, I'm not sure it will work fine in a Windows machine. You may need to play around a bit creating the environment from scratch. In the screenshot you uploaded, I can see one of the errors is actually related to the `model.h5` not being locally available. You need to download their pre-trained model and put it in the correct place. In addition to the repository that I've already linked, you can find some complementary instructions [here](https://github.com/cchamber/Infant_movement_assessment).

The `fidgetyfind` environment is separate from `openpose_keras` and you can create it from the commands mentioned before. You may need to run `conda deactivate` (to exit the `openpose_keras` environment) before creating the `fidgetyfind` environment.

If the above is still unclear let me know and I'll try to explain in more detail.

**Comment by msabers** [comment]
> Hi msabers,
> 
> Just to clarify, the `openpose_keras` and `fidgetyfind` are two separate environments. If you try to install the packages in both you will likely have conflicts.
> 
> For the `openpose_keras` environment, I'm not sure it will work fine in a Windows machine. You may need to play around a bit creating the environment from scratch. In the screenshot you uploaded, I can see one of the errors is actually related to the `model.h5` not being locally available. You need to download their pre-trained model and put it in the correct place. In addition to the repository that I've already linked, you can find some complementary instructions [here](https://github.com/cchamber/Infant_movement_assessment).
> 
> The `fidgetyfind` environment is separate from `openpose_keras` and you can create it from the commands mentioned before. You may need to run `conda deactivate` (to exit the `openpose_keras` environment) before creating the `fidgetyfind` environment.
> 
> If the above is still unclear let me know and I'll try to explain in more detail.

Thanks for your response and the clarification. 
First I need to deal with OpenPose_keras then run the FidgityFind.

**Comment by RomeroBarata** [comment]
Exactly, `openpose_keras` is separate to `fidgetyfind` code. But, you need to run `openpose_keras` first on your video to get the skeleton key points for use with the (`fidgetyfind`) code I provide here.
